### ISSUE TITLE
Lemma 2.9.6 Ambiguous / confusing language

### ISSUE THREAD

**Issue by iblumenfeld** [HoTT/book#913]
The language in the second part of Lemma 2.9.6 seems to be saying that we should have:

```
happly(q,p_{*}(a)) = g(p_{*}(a)).
```

Since `happly(q,p_{*}(a))` is of type

```
(p_{*}(f)(p_{*}(a)) = g(p_{*}(a))),
```

this doesn't make sense.  Perhaps there is a clearer way to state this Lemma using transport that doesn't give this confusion.

**Comment by mikeshulman** [comment]
What it says is that `happly(q,p_{*}(a))` (with its type specified, namely `(p_{*}(f)(p_{*}(a)) = g(p_{*}(a)))` as you say) is equal to a certain composite of other paths.  Why does that seem to be saying that `happly(q,p_{*}(a))` equals `g(p_{*}(a))`?

**Comment by iblumenfeld** [comment]
I'm not precisely sure why it confused me at first, but not only myself but several others who are working through the book with me have reported to me the same initial confusion at this point.  I think the issue is that it reads " `happy(q,p_{*}(a)) ` ...  is equal to the composite ..." and then gives a chain of equalities that ends with `g(p_{*}(a))`.  I think that it's the fact that it says "is equal to" and then the final thing in the sentence is ` g(p_{*}(a))` is what is causing the confusion. 

Does that help explain it at all?  I fear I am being just as confusing as I was confused.

**Comment by mikeshulman** [comment]
Hmm, I think I see what you're saying.  In set-based mathematics, we would never say that something is equal to *an equality*, so if someone wrote "A is equal to" and then a chain of equalities B=C=D=E, the only thing they could mean is that A is equal to B, C, D, and E, rather than equal to the composite *equality* between B and E.  And I guess the fact that we added "equal to *the composite*" (which, I guess, should really be "the concatenation", since I think we were trying to avoid the word "composite" for concatenation of equalities) wasn't enough to overcome this instinctive assumption, especially this early in the book.  So maybe some extra emphasis or clarification would be a good idea.  Any suggestions?

**Comment by iblumenfeld** [comment]
Pardon typos, I'm doing this on my phone, but I didn't want to just look like I was ignoring your request for too long:

I think one could clarify this by perhaps naming the paths composed. Maybe something like "is equal to the composition `i \cdot j \cdot k` where `i : (p_{*}(f))(p_{*}(a) = p_{*}(f(p^{-1}_{*}(p_{*}(a))))` is the path from in 2.9.4, j is ..."

It's very wordy, but maybe that's ok?

**Comment by mikeshulman** [comment]
Hmm.  We do use this style of writing elsewhere in the book as well, although I guess this is the first place it occurs; so we ought to find something to write here that will clue the reader permanently in so that they won't be confused when it happens again.  What if we take your suggestion and then follow it with an explanation about how we can write this more simply, without giving names to the intermediate paths, as a chain of equalities, and in the future we will frequently do so?

**Comment by peterlefanulumsdaine** [comment]
Closed by ed074a4
